docker-compose.yml

```diff
@@ -24,27 +24,28 @@ services:
 
   auth:
     container_name: auth
-    image: ghcr.io/neicnordic/sensitive-data-archive:v0.3.25-auth
+    command: sda-auth
+    image: ghcr.io/neicnordic/sensitive-data-archive:v0.3.25
     depends_on:
       credentials:
         condition: service_completed_successfully
     environment:
-      - ELIXIR_ID=${auth_ELIXIR_ID}
-      - ELIXIR_PROVIDER=http://${DOCKERHOST:-localhost}:8080/oidc/
-      - ELIXIR_SECRET=${auth_ELIXIR_SECRET}
-      - ELIXIR_JWKPATH=jwk
-      - ELIXIR_REDIRECTURL=http://localhost:8085/elixir/login
-      - LOG_LEVEL=info
-      - PUBLICFILE=/shared/c4gh.pub.pem
-      - RESIGNJWT=false
-      - S3INBOX=http://localhost:8000
+      - OIDC_ID=${auth_ELIXIR_ID}
+      - OIDC_PROVIDER=http://${DOCKERHOST:-localhost}:8080/oidc/
+      - OIDC_SECRET=${auth_ELIXIR_SECRET}
+      - OIDC_JWKPATH=jwk
+      - OIDC_REDIRECTURL=http://localhost:8085/oidc/login
       - LOG_LEVEL=debug
+      - RESIGNJWT=false
     extra_hosts:
       - ${DOCKERHOST:-localhost}:host-gateway
     volumes:
       - shared:/shared
+      - ${CONFIG_FILEPATH}:/config.yaml
     ports:
       - 8085:8080
+    networks:
+      - my-app-network
```

Comment:
Didn't we refactor stuff to not need this if the app had extra_hosts: specified so it could connect through the host network instead over the Docker network?

Reply:
oh, yes... we did. And now when testing again, it does works without adding `my-app-network`. I assume we can revert this change, or does @nanjiangshu have any objections?

Reply:
True, it's not needed. I have removed it in the commit 609ed18